e2e/config.js

```diff
@@ -4,5 +4,20 @@ module.exports = {
   solicitorUser: {
     password: defaultPassword,
     email: 'solicitor@example.com'
+  },
+  definition: {
+    jurisdiction: 'CMC',
```

Comment:
Isn't this `Civil` now?

Reply:
interesting one -> I think the label displayed in CCD will change, I don't think the internal code will. One to raise?

Reply:
Please double check, I will personally prefer to change that along with label to avoid the future confusion.